pkg/cmd/codespace/ssh.go

```diff
@@ -216,25 +218,31 @@ func (a *App) SSH(ctx context.Context, sshArgs []string, opts sshOptions) (err e
 	}
 }
 
-func shouldUseAutomaticSSHKeys(args []string, opts sshOptions) bool {
-	if opts.profile != "" {
-		// The profile may specify the identity file so cautiously don't override anything with that option
-		return false
-	}
+func shouldUseAutomaticSSHKeys(
```

Comment:
super subjective naming nit: `useAutomaticSSHKeys` - the function signature returning a `bool` already implies that predicate like function.

Reply:
Done